app/models/course.rb

```diff
@@ -2,4 +2,20 @@ class Course < ApplicationRecord
   def studying_for_headship?
     name == "NPQ for Headship (NPQH)"
   end
+
+  def npqh?
+    name == "NPQ for Headship (NPQH)"
+  end
+
+  def studying_for_aso?
+    name == "Additional Support Offer for new headteachers"
+  end
+
+  def aso?
+    name == "Additional Support Offer for new headteachers"
+  end
```

Comment:
Ditto here with alias

Reply:
I've removed all the duplicated methods and consolidated. so everything now goes through `course.npqh?` or `course.aso?`